lib/BaseController.js

```diff
@@ -127,7 +127,7 @@ module.exports = class BaseController {
   }
 
   errorHandler(err) {
-    if (typeof err === 'object') {
+    if (typeof err === 'object' && !(err instanceof Error)) {
```

Comment:
I think the instanceof check answers both checks in this case

if its not an instance of an error its an object...

The type of here can be a little misleading. `null` would be true here.

Reply:
there are some cases in the code where we Promise.reject({message: 'errmsg'}). the goal is to catch those objects and stringify them.